source/Octopus.Tentacle.Contracts/ScriptServiceV3Alpha/GetStatusCommandV3Alpha.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+
+namespace Octopus.Tentacle.Contracts.ScriptServiceV3Alpha
+{
+    public class GetStatusCommandV3Alpha
```

Comment:
Do these classes need to be tied to what is going over the wire, maybe it could just be `GetStatusCommand` no v3.

Reply:
They don't need to be. I wasn't sure what convention we were following.

I guess it depends whether we want the versioning of all commands to increment together (with `StartScriptCommandV3Alpha` etc.), or whether they can progress at their own cadence. 

I leant towards keeping them in sync, as we tend to make orchestrators (and now executors) for a specific version.

But if this is wrong, then I am happy to change it.

Reply:
We probably need more guidance.

But in general the tentacle client should be working as hard as it can be for you. Which I think should me trying to hide away details best it can, including new different wire command/responses.